libs/design/scss/typography/utilities/_variables.scss

```diff
@@ -0,0 +1,35 @@
+@use '../mixins/text-transform' as *;
+@use '../mixins/font-weight' as *;
+@use '../mixins/text-truncate' as *;
+
+$body-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica,
+	Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji';
+$base-font-family: $body-font-family;
+$monospace-font-family: ui-monospace, 'SFMono-Regular', SF Mono, Menlo, Consolas,
+	'Liberation Mono', monospace;
+
+// deprecated
+$desktop-font-size: 16px;
+$mobile-font-size: 16px;
+
+$large-font-size: 1.5rem;
+$medium-font-size: 1.25rem;
+$normal-font-size: 1rem;
+$small-font-size: 0.875rem;
+
+.embolden { /* stylelint-disable-line selector-class-pattern */
```

Comment:
These classes should not be in this file. Won't this cause an unexpected addition of CSS to the resulting bundle?

Reply:
I don't think so? They won't show up unless they are used.